node/src/components/contract_runtime.rs

```diff
@@ -78,6 +78,10 @@ pub struct ContractRuntimeMetrics {
     run_query: Histogram,
     get_balance: Histogram,
     get_validator_weights: Histogram,
+    get_era_validators_timing: Histogram,
+    get_era_validator_weights_by_era_id_timing: Histogram,
+    get_era_validators: Counter,
+    get_era_validator_weights_by_era_id: Counter,
```

Comment:
We don't need counters if we have the histogram.

Reply:
Removed counters